Website/src/rendering/src/stories/Pages/HomePage.stories.tsx

```diff
@@ -186,7 +188,7 @@ const Template: ComponentStory<typeof HeroSection> = () => {
         <HeaderCdpMessageBar />
         <HeroSection {...heroProps} />
         <ThreeColumnsSection {...threeColProps} />
-        <SponsorsGrid {...sponsorProps} />
+        <SelectedSponsorsGrid {...sponsorProps} />
```

Comment:
Please check the Azure DevOps build logs for the "Lint and Build Website" job. There is a TypeScript type mismatch in this Storybook story for this `sponsorProps` variable. Sometimes, this is easily fixed by casting the object to `unknown` before casting it the right type like this:
```
const sponsorProps = {
  // Content of the object
} as unknown as SponsorsProps;
```

You will see this trick being used a couple of places in our codebase.